docs/data/toolpad/connecting-to-data-sources/postgres.md

```diff
@@ -0,0 +1,25 @@
+# Postgres
+
+<p class="description">Postgres query type allows us to fetch data straight from Postgres database</p>
+
+## Working with Postgres
+
+As explained in the [connections](/toolpad/connecting-to-data-sources/connections/) section before you can use `Postgres` query you will first need to setup `connection`
+
+1. Once your `connection` is ready click on `ADD QUERY` in the `Properties editor` on the right
+
+1. Select `Postgres` type and click `CREATE QUERY`
+
+   ![Postgres type](/static/toolpad/postgres-query-1.png)
+
+1. We got all the basic configuration settings as described in [queries](/toolpad/connecting-to-data-sources/queries/) section
+
+1. In addition we can configure following properties inline
+
+   ![Postgres configuration](/static/toolpad/postgres-query-2.png)
+
+   - On the left side we have a textarea where we can pass `SQL` query
```

Comment:
I feel like we should mention how you can pass parameters here. 

I wonder if there could be a section about how state from the page can be used in queries in general through the parameters interface?